webapp/components/PostCard/index.vue

```diff
@@ -74,11 +82,16 @@ export default {
     HcRibbon,
     ContentMenu,
   },
+  mixins: [PostMutationHelpers],
   props: {
     post: {
       type: Object,
       required: true,
     },
+    width: {
+      type: Object,
+      default: () => {},
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-03T13:00:29Z" title="Monday, June 3rd 2019, 3:00:29 pm +02:00">Jun 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -74,11 +82,16 @@ export default {
     HcRibbon,
     ContentMenu,
   },
+  mixins: [PostMutationHelpers],
   props: {
     post: {
       type: Object,
       required: true,
     },
+    width: {
+      type: Object,
+      default: () => {},
```


Oh, ok, so `() => {}` creates an empty object. So, I guess the syntax for empty function is `() => () => {}` :thinking: